_data/simple-icons.json

```
@@ -15,6 +15,11 @@
             "hex": "00A98F",
             "source": "https://about.me/assets"
         },
+        {
+            "title": "ABB Robotstudio",
+            "hex": "EC1B24",
+            "source": "https://new.abb.com/products/robotics/de/robotstudio/downloads"
```

Where on this page can I find the logo?

Also, it would be nice to have the English link because this is an international project.

```suggestion
            "source": "https://new.abb.com/products/robotics/en/robotstudio/downloads"
```

---

ABB is very reluctant to show _any_ logos anywhere.
In the end the logo was taken from the `.exe`...